packages/python-dateutil/python-dateutil.spec

```diff
@@ -29,7 +29,7 @@ BuildRequires:  %{?scl_prefix}python%{python3_pkgversion}-six >= 1.5
 Summary:        %{summary}
 %{?python_provide:%python_provide python%{python3_pkgversion}-%{srcname}}
 Requires:       %{?scl_prefix}python%{python3_pkgversion}-six >= 1.5
-Obsoletes:      %{?scl_prefix}python36-dateutil
+Obsoletes:      %{?scl_prefix}python36-dateutil < %{version}-%{release}
```

Comment:
@evgeni My main curiosity is if this was done "on purpose".  I'm not sure yet if we can get downstream package tooling to accept this.  It currently rejects the non-versioned format.

Reply:
This comes from the original epel dateutil package and can be dropped.

Reply:
switched up to drop the obsolete all together